Dockerfile

```
@@ -0,0 +1,21 @@
+# Python version
+FROM python:3.7-alpine
+
+# Set environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+RUN apk update && apk add build-base postgresql-dev
+
+# Set work directory
+WORKDIR /code
+
+# Copy files to working directory
+COPY . /code/
+
+# Install dependencies
+RUN pip install pipenv
+RUN pipenv update
+RUN pipenv install
+
+ENTRYPOINT ["sh", "docker-entrypoint.sh"]
```

I think this could be:

```suggestion
ENTRYPOINT ["pipenv", "run", "python", "/src/manage.py", "runserver", "0.0.0.0:8000"]
```

---

My bad, this should be a `CMD`, not an `ENTRYPOINT`.